group_vars/model_tplink_eap225_outdoor_v1.yml

```diff
@@ -1,13 +1,9 @@
 ---
-imagebuilder: "https://downloads.openwrt.org/releases/21.02-SNAPSHOT/targets/ath79/generic/openwrt-imagebuilder-21.02-SNAPSHOT-ath79-generic.Linux-x86_64.tar.xz"
+imagebuilder: "https://downloads.openwrt.org/snapshots/targets/ath79/generic/openwrt-imagebuilder-ath79-generic.Linux-x86_64.tar.xz"
 feed_version: "1.2.0-snapshot"
```

Comment:
you should switch that to `master` too, to get the packagefeed thats for snapshot.

Reply:
There is a master feed? Did not know that.

Reply:
Oh, I miss-spelled that one. It's called `snapshot`, not master. But yes, it was there since the very first days.
1.2.0-snapshot is build with the 21.02-SDK, the snapshot feed with the snapshot one. You should defnitely go with the snapshot-feed, if you are going to use snapshot-imagebuilders.

https://firmware.berlin.freifunk.net/feed/snapshot/

Reply:
thanks I will try!